packages/node_modules/@ciscospark/react-component-people-list/src/index.js

```diff
@@ -1,6 +1,9 @@
+import 'react-virtualized/styles.css';
+
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import {List, AutoSizer} from 'react-virtualized';
```

Comment:
How much bigger does virtualized make this component? If it's significant, this change should probably be localized to the widget since not every time a people list is used virtualizating is necessary.

Reply:
The amount of work making two different virtualized vs non-virtualized scrolling components was rather large since @moriahmaney is handling the avatar loading only during scrolling.